syntax = "proto3";  //声明了protobuf的版本

package lddxfy; //声明了代码所在包

//定义下面的选项，表示生成service服务类和rpc方法的描述，默认不生成
option cc_generic_services = true;


message ResultCode
{
    int32 errcode = 1;
    bytes errmsg = 2;
}

message GetFriendListRequest
{
    uint32 userid = 1;
}

message GetFriendListResponse
{
    ResultCode res = 1;
    repeated bytes friends = 2;
}

service FriendServiceRPC
{
    
    rpc GetFriendList (GetFriendListRequest) returns (GetFriendListResponse);
}